src/input/TouchScreenImpl1.cpp

```diff
@@ -55,11 +55,11 @@ void TouchScreenImpl1::onEvent(const TouchEvent &event)
 
     switch (event.touchEvent) {
     case TOUCH_ACTION_LEFT: {
-        e.inputEvent = static_cast<char>(meshtastic_ModuleConfig_CannedMessageConfig_InputEventChar_RIGHT);
+        e.inputEvent = static_cast<char>(meshtastic_ModuleConfig_CannedMessageConfig_InputEventChar_LEFT);
```

Comment:
This may upset other implementations. can i have @mverch67 opinion on this?

Reply:
It may affect other platforms, but this seems to be a bug when tested. TOUCH_ACTION_LEFT should swipe from right to left.

Reply:
There is a reason for the original code. If you swipe from right to left the TOUCH_ACTION_LEFT is generated. If you swipe a screen from right to left then the screen moves to the left, disappears and the screen on the right reveals. So the trigger InputEventChar_RIGHT as in the original code is correct.

Reply:
Thank you. I see what you mean.